pkg/agents/team5/utils.go

```diff
@@ -10,11 +10,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func PercentageHP(a *CustomAgent5) int {
-	return int(float64(a.HP()) / float64(a.HealthInfo().MaxHP) * 100.0)
-}
-
-func (a *CustomAgent5) restrictToRange(lowerBound, upperBound, num int) int {
+func restrictToRange(lowerBound, upperBound, num int) int {
```

Comment:
Might want a valid range check on this function - feel free to ignore and resolve if you think it's fine